modules/imgimport/CMakeLists.txt

```diff
@@ -1,2 +1,2 @@
-include_directories(include ../core/include)
+include_directories(include ../core/include ../core/src)
```

Comment:
Like this?
include_directories(include ../core/include)
target_link_libraries(ImageImport ${OpenCV_LIBS} ${Boost_LIBRARIES} Core)
add_library(ImageImport src/imgimport.cpp)